neonvm-runner/cmd/main.go

```diff
@@ -1812,3 +1842,33 @@ func overlayNetwork(iface string) (mac.MAC, error) {
 
 	return mac, nil
 }
+
+func setNeonvmDaemonCPU(cpu vmv1.MilliCPU) error {
```

Comment:
both options (direct call to vm daemon or through runner) works for me and I don't have strong opinion. @sharnoff I remember you had some points, may I ask you to share them here?

Reply:
Yeah, broadly my view is that neonvm-daemon is privileged inside the VM, and we shouldn't expose it beyond the pod. With neonvm-runner, it should be _relatively_ easy for us to add mTLS to verify that connections are coming from neonvm-controller, but I'm hesitant to do that in neonvm-daemon because getting secrets into VMs is a little convoluted and has potentially unexpected properties (e.g., they won't be refreshed).

Similarly, in the worst-case scenario of a DoS attack on the HTTP server, I would rather that happens to neonvm-runner than neonvm-daemon so that the internals of the VM are unaffected. There's occasionally attacks on HTTP implementations that require unusual traffic, and having neonvm-runner act as an intermediary guards against that.

Reply:
Actually I just realized that even from perspective of the ownership going through runner is a bit more clear model. Let's say theoretically some operations can be performed only inside vm (throug daemon) but some only outside (through qmp). Why should controller bother to know the details what to use in every particular case?

Reply:
> Why should controller bother to know the details what to use in every particular case?

Great point, yeah. This is exactly the issue with having the controller directly connect to QEMU for QMP.